source/winKernel.py

```diff
@@ -479,3 +479,14 @@ def SetThreadExecutionState(esFlags):
 	if not res:
 		raise WinError()
 	return res
+
+
+def LCIDToLocaleName(windowsLCID: LCID) -> Optional[str]:
+	bufSize = 32
+	buf = ctypes.create_unicode_buffer(bufSize)
+	dwFlags = 0
+	try:
+		kernel32.LCIDToLocaleName(windowsLCID, buf, bufSize, dwFlags)
+	except AttributeError:
+		return None

```

Comment:
Do you know if this is an expected exception, if so please add a comment about why it is raised and harmless, otherwise add a log message so we can see when this happens.

I see that the code previously caught and `pass`ed, I would log at error level. Then we'll find out when this occurs and can decide at that point.

Reply:
I couldn't raise this exception.
When checking LCID values unknown to Windows, an empty buffer is returned.

Reply:
For completeness, I'll mention here that AttributeError on the kernel32 object would only be raised if LCIDToLocaleName did not exist in kernel32.dll. This function was added in Windows Vista, and because our minimum OS is Windows 7, this exception will never be raised, and thus there is no need to specifically handle it.